File: android/app/src/main/kotlin/com/example/pigeon_flutter_android/StartActivity.java
//package com.example.pigeon_flutter_android;
//
//import android.app.Activity;
//import android.os.Bundle;
//
//import com.example.pigeon_flutter_android.Pigeon.*;
//
//public class StartActivity extends Activity {
//    private class MyApi implements Api {
//        public Pigeon.SearchReply search(Pigeon.SearchRequest request) {
//            Pigeon.SearchReply reply = new Pigeon.SearchReply();
//            reply.result = String.format("Hi %s!", request.query);
//            return reply;
//        }
//    }
//
//    protected void onCreate(Bundle savedInstanceState) {
//        super.onCreate(savedInstanceState);
//        Pigeon.SetupApi(getBinaryMessenger(), new MyApi());
//    }
//}

File: android/app/src/main/kotlin/com/example/pigeon_flutter_android/StartActivity.kt
//package com.example.pigeon_flutter_android
//
//import android.app.Activity
//import dev.flutter.pigeon.Pigeon.Api
//import dev.flutter.pigeon.Pigeon.SearchRequest
//import dev.flutter.pigeon.Pigeon.SearchReply
//import android.os.Bundle
//import dev.flutter.pigeon.Pigeon
//import com.example.pigeon_flutter_android.StartActivity.MyApi
//
//class StartActivity : Activity() {
//    private inner class MyApi : Api {
//        fun search(request: SearchRequest): SearchReply {
//            val reply = SearchReply()
//            reply.result = String.format("Hi %s!", request.query)
//            return reply
//        }
//    }
//
//    override fun onCreate(savedInstanceState: Bundle?) {
//        super.onCreate(savedInstanceState)
//        Pigeon.SetupApi(getBinaryMessenger(), MyApi())
//    }
//}